dbt/adapters/databricks/connections.py

```diff
@@ -256,7 +258,8 @@ def open(cls, connection: Connection) -> Connection:
                 cls.validate_creds(creds, required_fields)
 
                 dbt_databricks_version = __version__.version
-                user_agent_entry = f"dbt-databricks/{dbt_databricks_version}"
+                dbt_invocation_env = os.getenv(DBT_INVOCATION_ENV) or "manual"
+                user_agent_entry = f"dbt-databricks/{dbt_databricks_version} ({dbt_invocation_env})"
```

Comment:
Looks like in dbt-spark we don't have the parentheses around `dbt_invocation_env`.

Reply:
Actually dbt-spark doesn't have this parameter (yet?).